DependencyInjection/CoopTilleulsMigrationExtension.php

```diff
@@ -35,6 +37,13 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('coop_tilleuls_migration.legacy_connection_name', $config['legacy_connection_name']);
 
+        if (method_exists($container, 'registerForAutoconfiguration')) {
```

Comment:
This is useless with `symfony/dependency-injection: ^3.3` in your `composer.json` file.